src/llmcompressor/pipelines/registry.py

```diff
@@ -75,12 +76,7 @@ def _validate_infer_pipeline(modifiers: List[Modifier]) -> str:
             quant_modifier = active_qmods[0]
             config = quant_modifier.resolve_quantization_config()
             if config.requires_calibration_data():
-                return "basic"
-            else:
-                return "datafree"
-
-        if any(isinstance(modifier, SmoothQuantModifier) for modifier in modifiers):
-            return "basic"
+                return "sequential"
```

Comment:
So datafree pipeline, which is not sequenitally onloaded, should be just for the QuantizationModifier at this point, when doing weight-only quantization?

Reply:
Yes, by nature of the fact that that is the only configuration where `modifier not in CALIBRATION_MODIFIERS` and `not config.requires_calibration_data()`

Reply:
I've simplified this logic to make this clearer